src/Orc.Automation.Tests/ScenarioManagement/Loggers/UserDocumentationBuilderLogger.cs

```diff
@@ -0,0 +1,175 @@
+        var afterImagePath = Path.Combine(GetScenarioDirectory(currentScenario), $"after_{currentStepIndex}.jpg");
+        bitmap?.Save(afterImagePath, ImageFormat.Jpeg);
+    }
+
+    public void LogBeforeUserInteraction(UserInteraction userInteraction)
+    {
+    }
+
+    public void LogAfterUserInteraction(UserInteraction userInteraction)
+    {
+    }
+
+    public void LogScenarioFinish(AutomationScenario scenario)
+    {
+        var htmlStringBuilder = new StringBuilder();
+
+        htmlStringBuilder.AppendLine("<!DOCTYPE html><html style=\"font-family:verdana\"><head><meta name=\"viewport\" content=\"width=device-width, initial-scale=1\">");
+        htmlStringBuilder.AppendLine("<style>");
+        htmlStringBuilder.AppendLine("* {box-sizing: border-box;}");
+        htmlStringBuilder.AppendLine(".column {  float: left;  width: 33.33%;  padding: 10px;}");
+        htmlStringBuilder.AppendLine(".column1 {  float: left;  width: 20%;  padding: 10px;}");
+        htmlStringBuilder.AppendLine(".row:after {  content: \"\";  display: table;  clear: both;}");
+        htmlStringBuilder.AppendLine("</style>");
+        htmlStringBuilder.AppendLine("</head><body>");
+        htmlStringBuilder.AppendLine($"<h1>{scenario.Name}</h1>");
+        htmlStringBuilder.AppendLine($"<h2>{scenario.Description}</h2>");
+        htmlStringBuilder.AppendLine($"<div class=\"row\"><div class=\"column1\" style=\"background-color:#ccc;\"><h2>Step</h2></div><div class=\"column\" style=\"background-color:#ccc;\"><h2>Before</h2></div><div class=\"column\" style=\"background-color:#ccc;\"><h2>After</h2></div></div>");
+
+        for (var i = 0; i < scenario.StartedSteps.Count; i++)
+        {
+            var step = scenario.StartedSteps[i];
+            htmlStringBuilder.AppendLine($"<div class=\"row\"><div class=\"column1\"><h4>{step.Name}</h4></div><div class=\"column\"><img src=\"before_{i}.jpg\" style=\"object-fit:cover;width:100%;height:100%;border: solid 1px #CCC\"/></div><div class=\"column\"><img src=\"after_{i}.jpg\" style=\"object-fit:cover;width:100%;height:100%;border: solid 1px #CCC\"/></div></div>\r\n");
+        }
+
+        htmlStringBuilder.AppendLine("</body></html>");
+
+        var indexHtmlPath = Path.Combine(GetScenarioDirectory(scenario), "Index.html");
+        File.WriteAllText(indexHtmlPath, htmlStringBuilder.ToString());
+    }
+
+    private Bitmap? ScreenShot(Rect bounds)
+    {
+        try
+        {
+            Wait.UntilResponsive(200);
+
+            var captureBitmap = new Bitmap((int)bounds.Width, (int)bounds.Height, PixelFormat.Format32bppArgb);
+            var captureGraphics = Graphics.FromImage(captureBitmap);
+            captureGraphics.CopyFromScreen((int)bounds.X, (int)bounds.Y, 0, 0, new System.Drawing.Size((int)bounds.Width, (int)bounds.Height));
+
+            captureGraphics.Dispose();
+
+            return captureBitmap;
+        }
+        catch (Exception ex)
+        {
+            Console.Write(ex.ToString());
```

Comment:
Write to console? I would like this to be in the results file as well

Reply:
will write to test output